pyproject.toml

```diff
@@ -1,6 +1,20 @@
 [build-system]
-requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
-build-backend = "poetry.core.masonry.api"
+requires = ["maturin>=0.14.8,<0.15", "poetry-dynamic-versioning"]
+build-backend = "maturin"
```

Comment:
This one might be the most delicate change, since it is actually affecting how we build the package.

However, we should just test once making a pseudo-tag locally (that is a proper tag, but avoiding pushing it) and building with `pip wheel` (that should be what we should use in the workflow, but we are not).
https://github.com/NNPDF/workflows/blob/818e8d74406df7c51992ea3b9377441a401a7087/.github/workflows/python-poetry-pypi.yml#L66-L69

So, updating the workflow with `pip wheel` (that should keep working for Poetry as well) is a requirement for merging this.

Reply:
you're right - this is a delicate change and most likely we will break the package (even without touching the code); I suggest we do another `.patch` file

Reply:
But then I don't understand? how can we deal with the maturin backend without having maturin in game?

Reply:
without rust we don't need and want maturin as backend, no? (just to be clear: backends means the one who actually builds the wheel (creating the folder blabla))

Reply:
Without Rust we don't need `maturin`. But I see no reason why we don't want it, since it could work for both.

Reply:
postponed to #340